README.md

```diff
@@ -114,6 +114,21 @@ sent to the callback URL:
     "details" : "The file has a virus"
 }
 ```
+##Whitelisting client services
+Upscan supports configuration of supported file types on a per-service basis.
+
+Calling services must therefore identify themselves in requests via the `User-Agent` header.
+If the supplied value is not in Upscan's list of allowed services then the `/initiate` call will fail with a `403` error.
+
+In addition to returning a `403` error, Upscan will log details of the Forbidden request. For example:
+
+```json
+{"app":"upscan-initiate","message":"Invalid User-Agent: [Some(my-unknown-service-name)].","logger":"application","level":"WARN"}
+```
+
+To start the onboarding process for a new service, please contact the Upscan team via [#event-upscan](https://hmrcdigital.slack.com/messages/C8XPL559N/).
```

Comment:
Maybe we should ask teams to contact #team-plat-services here, as that will be the more general channel going forwards I think?